src/coreclr/jit/importer.cpp

```diff
@@ -11952,7 +11953,7 @@ void Compiler::impImportBlockCode(BasicBlock* block)
 
             ADRVAR:
 
-                op1 = gtNewLclvNode(lclNum, lvaGetActualType(lclNum) DEBUGARG(opcodeOffs + sz + 1));
+                op1 = impCreateLocalNode(lclNum DEBUGARG(opcodeOffs + sz + 1));
```

Comment:
For `ldarga` opcodes we were creating locals that require normalization with the wrong type here. This could be problematic since we sometimes optimize `GT_IND (GT_ADDR (GT_LCL_VAR ...)))` to `GT_LCL_VAR` very early during import, in which case we could have problems.